### ISSUE TITLE
Some other lut that are not working

### ISSUE THREAD

**Issue by stephane-archer** [JakubOchnik/Cube-LUT-Loader#17]
You can find below some other lut that are currently not working.
you mentioned that one of the previous LUT had values over 1.0 without a DOMAIN_MAX specified.
In my video editor, I have to manually select the input and output color space (Rec 709, Rec 2020), could this be linked to that? or the fact there are 8 bits and 10 bits video files?

[lutNotWorking.zip](https://github.com/JakubOchnik/Cube-LUT-Loader/files/12328493/lutNotWorking.zip)

**Comment by JakubOchnik** [comment]
Fixed - clipping LUT values outside the domain resolved the issue (tested the results for your .cubes and they seem correct).

**Comment by stephane-archer** [comment]
Thank you so much @JakubOchnik for taking the time to look at this issue. You are the best!
Does "clipping LUT values" mean ignoring the values outside the DOMAIN_MAX?
Have you looked at what other software is currently doing? Because these LUTs with values outside the DOMAIN_MAX seem to be everywhere on the internet. I wonder if the specs for .cube have evolved?

**Comment by JakubOchnik** [comment]
Hi, @stephane-archer. Sorry for replying so late.

By clipping I mean treating values outside of the range as min/max (e.g. if the value is 1.10, assume it's 1.0).

I know that LUTs with such values are common, but handling them is a mystery to me 😄 I tried looking for possible approaches in other open-source software, but the only one I found was the one I implemented. I guess that specifying the custom domain is just not a common practice in LUT generators...

However, I have another approach in mind - finding the min/max values and scaling the whole range back to 0-1. I'll try to implement it and compare the results in my free time.
